lib/gitarro/backend.rb

```diff
@@ -174,6 +171,20 @@ def magicword(pr_number, context)
 
   private
 
+  # Show a message stating if there are opened PRs or not
+  def print_pr_resume(prs)
+    if pr_list_empty?(prs)
+      if @options[:changed_since] >= 0
+        puts '[PRS=FALSE] No Pull Requests opened or with changes ' \
+             "newer than #{options[:changed_since]} seconds"
+      elsif @options[:changed_since] < 0
+        puts '[PRS=FALSE] No Pull Requests opened'
```

Comment:
i would create a real bool variable instead of hardcoding it into string
```ruby
 irb
irb(main):001:0> foo = true
=> true
irb(main):002:0> puts foo
true
=> nil
irb(main):003:0> puts "#{foo}"
true
=> nil
irb(main):004:0> puts "#{foo}gino"
truegino

```

Reply:
i think that we can just print/reuse

``` prs.any?`` which is true when full, otherwise false

Reply:
Agreed, but we cannot remove pr_list_empty since it is used at gitarro.rb

Reply:
yop io wouldnt remove it

Reply:
My mistake, we can remove it. We said that we will exit with 0 if pr list is empty, no matter what :-)